src/wallet/bdb.h

```diff
@@ -217,6 +217,7 @@ class BerkeleyBatch
     Db* pdb;
     std::string strFile;
     DbTxn* activeTxn;
+    Dbc* m_cursor;
```

Comment:
In commit "walletdb: Handle cursor internally" (84bc5c1705691f4ecdfb8bcc90eb1bc4970281dc)

Could the commit message for this say something about the motivation for this change? It seems good, but I'm also not sure I understand why it's happening.

Reply:
Done.

This is happening because `Dbc` is BDB specific so a generic `DatabaseBatch` class can't return them.